kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -1585,8 +1585,7 @@ public void setTypeMap(Map<String, Class<?>> map) throws SQLException {
 
   @Override
   public boolean isWrapperFor(Class<?> iface) throws SQLException {
-    // TODO Auto-generated method stub
-    throw new SQLFeatureNotSupportedException("Method not supported");
+    return iface.isInstance(this);
```

Comment:
what is the use case?

The implementation in this pr just likes isInstance and asInstance.

Reply:
I think it's a standard method, no matter whether it looks like isInstance and asInstance or not.